.circleci/config.yml

```diff
@@ -220,6 +220,31 @@ jobs:
               echo "Not pushing to dockerhub for tag=${CIRCLE_TAG} branch=${CIRCLE_BRANCH}"
             fi
 
+  rust-changelog:
+    docker:
+      - image: python:3.7-buster
+    steps:
+      - setup_remote_docker:
+          version: 18.02.0-ce
+      - run:
+          name: Install Docker client
+          command: |
+            set -x
+            VER="18.06.3-ce"
+            curl -L -o /tmp/docker-$VER.tgz https://download.docker.com/linux/static/stable/x86_64/docker-$VER.tgz
+            tar -xz -C /tmp -f /tmp/docker-$VER.tgz
+            mv /tmp/docker/* /usr/bin
+      - run:
+          name: pull find-package-rugaru image
+          command: |
+            docker pull gguthemoz/fpr:latest
```

Comment:
honestly, this makes me a tiny bit concerned. I frequently chase down folks and packages that are no longer maintained by one person. Is it possible to put this under a larger org so that someone else can take it over if need be?

Reply:
Absolutely, that was just to get something out the door. Let me get added to the mozilla docker org.